bindings/java/doxygen/schema/db_schema_8_6.dox

```diff
@@ -176,33 +184,37 @@ NOTE: This table is not used in any code.
 
 
 # Blackboard Tables 
-The \ref mod_bbpage is used to store results from analysis modules. 
+The \ref mod_bbpage "Blackboard" is used to store results from analysis modules. 
 
 ## blackboard_artifacts
 Stores artifacts associated with objects.
 - **artifact_id** - Id of the artifact (assigned by the database)
 - **obj_id** - Id of the associated object
 - **artifact_type_id** - Id for the type of artifact (can be looked up in the blackboard_artifact_types table)
+- **data_source_obj_id** - Id of the data source for this artifact
+- **artifact_type_id** - Type of artifact (references artifact_type_id in blackboard_artifact_types)
+- **review_status_id** - (references review_status_id in review_statuses)
```

Comment:
How about we standardize on referring to "the artifact" and use that where we currently sometimes have "this artifact" or just "artifact." I don't actually care what we choose, but consistency would be nice.

Reply:
Let's also save this for a future PR